inc/functions.php

```diff
@@ -220,6 +220,15 @@ function ml_wpsearch_search_query($query, $params)
         'options' => $options['rest_config_option'],
         'transform' => $options['rest_transform'],
     )));
+    
+    // send to different endpoint with empty search
+    if ( is_array( $new_params['options'] ) ) {
+        if ( empty( $query['search']['qtext'] ) ) {
```

Comment:
I just realized this line should be

`if ( empty( $query['search']['qtext'] ) && !empty( $new_params['options'][1] ) ) {`